ani-cli

```diff
@@ -310,12 +310,12 @@ download_dir="${ANI_CLI_DOWNLOAD_DIR:-.}"
 quality="${ANI_CLI_QUALITY:-best}"
 no_menu=0
 case "$(uname -a)" in
-    *Darwin*) player_function="${ANI_CLI_PLAYER:-iina}" ;;           # mac OS
-    *ndroid*) player_function="${ANI_CLI_PLAYER:-android_mpv}" ;;    # Android OS (termux)
-    *steamdeck*) player_function="${ANI_CLI_PLAYER:-flatpak_mpv}" ;; # steamdeck OS
-    *MINGW*) player_function="${ANI_CLI_PLAYER:-mpv.exe}" ;;         # Windows OS
-    *ish*) player_function="${ANI_CLI_PLAYER:-iSH}" ;;               # iOS (iSH)
-    *) player_function="${ANI_CLI_PLAYER:-mpv}" ;;                   # Linux OS
+    *Darwin*) player_function="${ANI_CLI_PLAYER:-iina}" ;;            # mac OS
+    *ndroid*) player_function="${ANI_CLI_PLAYER:-android_mpv}" ;;     # Android OS (termux)
+    *steamdeck*) player_function="${ANI_CLI_PLAYER:-flatpak_mpv}" ;;  # steamdeck OS
+    *MINGW* | *WSL2*) player_function="${ANI_CLI_PLAYER:-mpv.exe}" ;; # Windows OS
+    *ish*) player_function="${ANI_CLI_PLAYER:-iSH}" ;;                # iOS (iSH)
+    *) player_function="${ANI_CLI_PLAYER:-mpv}" ;;                    # Linux OS
```

Comment:
why the addition of spaces?

Reply:
There is no apparent reason for that. Does it need to be changed?

Reply:
yes, please make it as it was

Reply:
I found now reason for it(Syntax highlighting issue) but I can still change it:
![obraz](https://github.com/pystardust/ani-cli/assets/37329111/52d4bbc1-98ff-4c97-a673-257b9f1aa4cd)

Reply:
add a space after the `;` on the wsl line, and align all the comments to be like that please

Reply:
Then I don't understand what needs to be changed as in unchanged pull request file this spaces are added.

Reply:
ah ok, sorry it's phone formatting so i couldn't see clearly